src/main/java/com/minecolonies/coremod/blocks/MultiBlock.java

```diff
@@ -0,0 +1,146 @@
+ * Every solid block can be substituted by this block in schematics.
+ * This helps make schematics independent from location and ground.
+ */
+public class MultiBlock extends AbstractBlockMinecolonies<MultiBlock>
+{
+
+    /**
+     * The hardness this block has.
+     */
+    private static final float BLOCK_HARDNESS = 0.0F;
+
+    /**
+     * This blocks name.
+     */
+    private static final String BLOCK_NAME = "multiBlock";
+
+    /**
+     * The resistance this block has.
+     */
+    private static final float RESISTANCE = 1F;
+
+    /**
+     * Constructor for the Substitution block.
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    public MultiBlock()
+    {
+        super(Material.WOOD);
+        initBlock();
+    }
+
+    /**
+     * initialize the block
+     * sets the creative tab, as well as the resistance and the hardness.
+     */
+    private void initBlock()
+    {
+        setRegistryName(BLOCK_NAME);
+        setUnlocalizedName(String.format("%s.%s", Constants.MOD_ID.toLowerCase(Locale.ENGLISH), BLOCK_NAME));
+        setCreativeTab(ModCreativeTabs.MINECOLONIES);
+        setHardness(BLOCK_HARDNESS);
+        setResistance(RESISTANCE);
+    }
+
+    @Override
+    public boolean onBlockActivated(
+            final World worldIn,
+            final BlockPos pos,
+            final IBlockState state,
+            final EntityPlayer playerIn,
+            final EnumHand hand,
+            final EnumFacing facing,
+            final float hitX,
+            final float hitY,
+            final float hitZ)
+    {
+        if (worldIn.isRemote)
+        {
+            MineColonies.proxy.openMultiBlockWindow(pos);
+        }
+        return true;
+    }
+
+    @Override
+    public void neighborChanged(final IBlockState state, final World worldIn, final BlockPos pos, final Block blockIn, final BlockPos fromPos)
+    {
+        if(worldIn.isRemote)
+        {
+            return;
+        }
+        final TileEntity te = worldIn.getTileEntity(pos);
+        if(te instanceof TileEntityMultiBlock)
+        {
+            ((TileEntityMultiBlock) te).handleRedstone(worldIn.isBlockPowered(pos));
+        }
+    }
+
+    @Override
+    public boolean hasTileEntity(final IBlockState state)
+    {
+        return true;
+    }
+
+    @Override
+    public TileEntity createTileEntity(final World world, final IBlockState state)
+    {
+        return new TileEntityMultiBlock();
+    }
+
+    /**
+     * @deprecated (Remove this as soon as minecraft offers anything better).
+     */
+    @SuppressWarnings(DEPRECATION)
+    @Override
+    @Deprecated
+    public boolean isFullBlock(final IBlockState state)
+    {
+        return false;
+    }
+
+    /**
+     * Used to determine ambient occlusion and culling when rebuilding chunks
+     * for render.
+     *
+     * @return true
+     *
+     * @deprecated
+     */
+    //todo: remove once we no longer need to support this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)